packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/order-api/basic-usage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+  reactExtension,
+  Screen,
+  Navigator,
+  ScrollView,
+  Text,
+  Section,
+  useOrder,
+} from '@shopify/ui-extensions-react/point-of-sale';
+
+const SmartGridModal = () => {
+  // Use the useOrder hook to access order information
+  const order = useOrder();
+
+  return (
+    <Navigator>
+      <Screen name="OrderDetails" title="Order Details">
+        <ScrollView>
+          <Section title="Order Information">
+            <Text>Order ID: {order.id}</Text>
+            <Text>Order Name: {order.name}</Text>
+            {order.customerId && <Text>Customer ID: {order.customerId}</Text>}
+          </Section>
+        </ScrollView>
+      </Screen>
+    </Navigator>
+  );
+};
+
+export default reactExtension('pos.purchase.post.block.render', () => (
+  <SmartGridModal />
+));
```

Comment:
Would like to see the AI be able to tell this is supposed to be an action extension and therefore shouldn't be SmartGridModal